Number_types/include/CGAL/boost_mp.h

```diff
@@ -29,7 +29,7 @@
 // easy solution.
 // TODO: MSVC has trouble with versions <= 1.69, reenable once 1.70 has been
 // tested. https://github.com/boostorg/multiprecision/issues/98
-#if !defined CGAL_DO_NOT_USE_BOOST_MP && BOOST_VERSION >= 106300 && !defined _MSC_VER
+#if !defined CGAL_DISABLE_GMP && !defined CGAL_DO_NOT_USE_BOOST_MP && BOOST_VERSION >= 106300 && !defined _MSC_VER
```

Comment:
@lrineau Could you explain why?

Reply:
My guess is that we only had the GMP-backend in mind for `boost::mp`.

Reply:
> @lrineau Could you explain why?

From what I can see, at that time I only considered `boost::multiprecision::mpz_int`, and not `cpp_int`, or whatever other backend.

Maybe there was a platform of the testsuite with compilation errors with `cpp_int`. Now that CGAL-5.2-beta1 is almost done, I think we can experiment again in `integration`: we are at the very beginning of the release cycle for CGAL-5.3 (due in June 2021). I suggest we remove the `!defined CGAL_DISABLE_GMP &&` and see what happens in the test results.